### ISSUE TITLE
The case problem of the file: StackOverflowIcon.js

### ISSUE THREAD

**Issue by creamidea** [at-icons/material#4]
Hi, I have met this problem recently.
(My OS is Linux.)

`Module not found: Error: Cannot resolve 'file' or 'directory' ./StackOverflowIcon.js in @icons/material/index.js`

I found [some code](https://github.com/at-icons/material/blob/e6fcc0cfc8fdc09a4672bcda00cecb44adcebe71/icons/index.js#L1873) in icons/index.js 
```js
export { default as StackOverflowIcon } from './StackOverflowIcon.js'
```
But, there is no file named `StackOverflowIcon.js` which should be `StackoverflowIcon.js` in npm-package.
You can check this [npm-dist-link](https://registry.npmjs.org/@icons/material/-/material-0.2.4.tgz).

**Comment by gelashviligiorgi** [comment]
+1

**Comment by casesandberg** [comment]
If this is a high-priority issue for you, consider sending a PR to fix it.

**Comment by armenzg** [comment]
I'm rather surprised the version 0.3.0 has been out for so long and only now and issue was filed.
I don't have any issues locally, however, Travis cannot get over it:
https://travis-ci.org/mozilla/bugzilla-dashboard/jobs/514853519

I cannot reproduce the issue locally with `yarn build`.

To work around it I did the following:
```diff
-import { AccountCircleIcon, LoginIcon, LogoutIcon } from '@icons/material';
+import AccountCircleIcon from '@icons/material/AccountCircleIcon';
+import LoginIcon from '@icons/material/LoginIcon';
+import LogoutIcon from '@icons/material/LogoutIcon';
```

**Comment by casesandberg** [comment]
This has been fixed in the newest version `@icons/material@0.4.0`
